ui/vite.config.ts

```diff
@@ -0,0 +1,19 @@
+import react from "@vitejs/plugin-react-swc";
+import { defineConfig } from "vite";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  build: {
+    outDir: "dist/ui",
+  },
+  server: {
+    proxy: {
+      "/admin": {
+        target: "http://localhost:8000",
```

Comment:
Why is this hardcoded? Is it possible to put it in some .env?

Reply:
Fixed